db/migrate/20220818113551_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :users, null: false, foreign_key: true
+      t.references :posts, null: false, foreign_key: true
```

Comment:
hello there 👋🏾 , although you have done a fantastic job creating the table and populating it. there is still one tiny 👌🏾  detail left to address. 

you failed to reference the users table with its foreign key constraint `foreign_key : {to_table: another table}` .

Reply:
Hi, @alexander16108 thank you for your review.